src/js/components/CollapsiblePanelComponent.jsx

```diff
@@ -27,10 +28,16 @@ var CollapsiblePanelComponent = React.createClass({
       this.setState({
         isOpen: nextProps.isOpen
       });
+      if (this.props.togglePanel != null) {
+        this.props.togglePanel(false);
+      }
```

Comment:
Small thing, but the presence of this test seems rather unidiomatic to me - I'd prefer `props.togglePanel` to be required or no-op by default.